src/webhook_launcher/app/payload.py

```diff
@@ -129,138 +83,162 @@ def create_placeholder(self, repourl, branch, packages=None):
                                     comment="Placeholder")
             mapobj.save()
             mapobjs.append(mapobj)
-    
+
         return mapobjs
 
     def relay(self, relays=None):
 
         if not self.url:
-            print "Trying to relay but Payload has no url, skipping"
+            print("Trying to relay but Payload has no url, skipping")
             return
 
         parsed_url = urlparse.urlparse(self.url)
-        official_projects = list(set(prj.name for prj in Project.objects.filter(official=True, allowed=True)))
-        official_packages = list(set(mapobj.package for mapobj in
-                                WebHookMapping.objects.filter(repourl=self.url,
-                                project__in=official_projects).exclude(package="")))
+        official_projects = list(
+            set(prj.name for prj in
+                Project.objects.filter(official=True, allowed=True)
+                )
+        )
+        official_packages = list(
+            set(mapobj.package for mapobj in
```

Comment:
Cosmetic: I find this style where after the opening brace there is still stuff, `set(map`, but the opening brace on its own line a bit awkward. Add a newline and reflow?  Also the other occurences.